WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -3133,4 +3133,15 @@ private void updateAddingMediaToEditorProgressDialogState(ProgressDialogUiState
         mAddingMediaToEditorProgressDialog = mProgressDialogHelper
                 .updateProgressDialogState(this, mAddingMediaToEditorProgressDialog, uiState, mUiHelpers);
     }
+
+    @Override
+    public String onFailedToInsertMediaTapped(int mediaId) {
+        MediaModel media = mMediaStore.getMediaWithLocalId(mediaId);
+
+        if (media != null) {
+            UploadUtils.getErrorMessageFromMedia(this, media);
```

Comment:
I think you're missing a "return" here :-)

Reply:
🤦‍♂️...I just hope it's the today super hot temperature and not a permanent thing 🙃! I modified it locally but missed commit/push...thanks for catching 🙇‍♂️! changed here fe9518f07c15bcacf34e7d5f68d9113b41c79f66